presto/presto.sh

```diff
@@ -157,33 +151,30 @@ function configure_master(){
   else
     include_coordinator='false'
   fi
-  cat > presto-server-${PRESTO_VERSION}/etc/config.properties <<EOF
+  cat > presto-server-${STARBURST_PRESTO_VERSION}/etc/config.properties <<EOF
 coordinator=true
 node-scheduler.include-coordinator=${include_coordinator}
 http-server.http.port=${HTTP_PORT}
 query.max-memory=999TB
 query.max-memory-per-node=${PRESTO_QUERY_NODE_MB}MB
 query.max-total-memory-per-node=${PRESTO_QUERY_NODE_MB}MB
 memory.heap-headroom-per-node=${PRESTO_HEADROOM_NODE_MB}MB
-resources.reserved-system-memory=${PRESTO_RESERVED_SYSTEM_MB}MB
 discovery-server.enabled=true
 discovery.uri=http://${PRESTO_MASTER_FQDN}:${HTTP_PORT}
 EOF
-
   # Install cli
-  $(wget https://repo1.maven.org/maven2/com/facebook/presto/presto-cli/${PRESTO_VERSION}/presto-cli-${PRESTO_VERSION}-executable.jar -O /usr/bin/presto)
+  $(wget https://storage.googleapis.com/starburstdata/presto/${PRESTO_MAJOR_VERSION}e/${STARBURST_PRESTO_VERSION}/presto-cli-${STARBURST_PRESTO_VERSION}-executable.jar -O /usr/bin/presto)
   $(chmod a+x /usr/bin/presto)
 }
 
 function configure_worker(){
-  cat > presto-server-${PRESTO_VERSION}/etc/config.properties <<EOF
+  cat > presto-server-${STARBURST_PRESTO_VERSION}/etc/config.properties <<EOF
 coordinator=false
 http-server.http.port=${HTTP_PORT}
 query.max-memory=999TB
 query.max-memory-per-node=${PRESTO_QUERY_NODE_MB}MB
 query.max-total-memory-per-node=${PRESTO_QUERY_NODE_MB}MB
 memory.heap-headroom-per-node=${PRESTO_HEADROOM_NODE_MB}MB
```

Comment:
PRESTO_HEADROOM_NODE_MB is only 256MB. This is likely way too small.
By default, we reserve 30% of Xmx, so orders of magnitude more.

Reply:
I agree. I was just leaving as is. I think we should properly improve -- calculate_memory() to address all configs.